blackbox/docs/sql/statements/create-table.rst

```diff
@@ -426,10 +426,47 @@ By default data is stored using ``LZ4`` compression. This can be changed to
 ``best_compression`` which uses ``DEFLATE`` for a higher compression ratio, at
 the expense of slower column value lookups.
 
-
 :values:
   ``default`` or ``best_compression``
-  
+
+.. _table_parameter.store_type:
+
+``store.type``
+--------------
+
+The store type setting allows you to control how data is stored
+and accessed on disk. The following storage types are supported:
+
+:fs:
+  Default file system implementation. It will pick the best
+  implementation depending on the operating environment.
```

Comment:
why not write that it will be `hybridfs` on all supported systems like ES does? I think it is very interesting what the default is...

Reply:
sure, makes sense! pushed the fixup